### ISSUE TITLE
Only one peer for NAT nodes

### ISSUE THREAD

**Issue by forchain** [Qitmeer/qitmeer#390]
#### Synchronization is done 

```sh
2020-11-10|09:45:54.291 [INFO ] Your synchronization has been completed. module=SYNC

```
### Peer info
```
Outlier@Mac ~/github.com/qitmeer/qitmeer/script (0.10-dev*) $ ./cli.sh -notls -p 28131 peerinfo

[
  {
    "id": "16Uiu2HAmHoq3MqSt7YyGeqr5b5eq3aeYdFE8UkJRT9d1YFVZwPeF",
    "qnr": "qnr:-Iu4QD3H-HvryfLH83WSxlhBf5zsrKSsJcvqR_iFdJ_u1-4XNXwz7mQ-VLStBt8-SJIox5wU-OuCN01sL46jHsM6v2sBgmlkgnY0gmlwhJDKWkGJc2VjcDI1NmsxoQNMjiLTk_DxxB8CHyYo5fmCqpI0t3vYQKyP0xaPzobieIN0Y3CCbeKDdWRwgm3s",
    "state": "disconnected",
    "graphstate": {
      "tips": null,
      "mainorder": 0,
      "mainheight": 0,
      "layer": 0
    },
    "timeoffset": 0
  }
]
```

P.S. public nodes are fine

**Comment by dindinw** [comment]
@forchain Please confirm if it fixed by `0.10.0+dev-80adeac`

**Comment by forchain** [comment]
verified.

Working both on Symmetric NAT and Restrict NAT
